<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>telaSCRIPTS</class>
 <widget class="QMainWindow" name="telaSCRIPTS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MCTY_FACIL</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>280</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>280</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btn_pmwin10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:1px solid rgb(0,0,0)</string>
       </property>
       <property name="text">
        <string>script win10</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_pmwin7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:1px solid rgb(0,0,0)</string>
       </property>
       <property name="text">
        <string>script win7</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Pague Menos</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_sicredwin7">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>50</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:1px solid rgb(0,0,0)</string>
       </property>
       <property name="text">
        <string>script win7</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>213</x>
         <y>20</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>SICRED</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_pifpafwin7">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>50</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:1px solid rgb(0,0,0)</string>
       </property>
       <property name="text">
        <string>script win7</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>20</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Rio Branco</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_pifpafwin10">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>100</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:1px solid rgb(0,0,0)</string>
       </property>
       <property name="text">
        <string>script win10</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_voltar">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>20</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:1px solid rgb(0,0,0)</string>
       </property>
       <property name="text">
        <string>Voltar</string>
       </property>
      </widget>
      <zorder>label_4</zorder>
      <zorder>label_3</zorder>
      <zorder>label_2</zorder>
      <zorder>btn_pmwin10</zorder>
      <zorder>btn_pmwin7</zorder>
      <zorder>btn_sicredwin7</zorder>
      <zorder>btn_pifpafwin7</zorder>
      <zorder>btn_pifpafwin10</zorder>
      <zorder>btn_voltar</zorder>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>320</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>320</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 183, 2);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>740</x>
         <y>290</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>By:Adjair</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="imagem microcity/imagens.qrc"/>
 </resources>
 <connections/>
</ui>
